api/lib/domain/services/authentication/pole-emploi-authentication-service.js

```diff
@@ -5,17 +5,25 @@ const querystring = require('querystring');
 const { AuthenticationTokenRetrievalError } = require('../../errors');
 const AuthenticationSessionContent = require('../../models/AuthenticationSessionContent');
 const jsonwebtoken = require('jsonwebtoken');
-const { POLE_EMPLOI } = require('../../constants').SOURCE;
+const constants = require('../../constants');
 
 const DomainTransaction = require('../../../infrastructure/DomainTransaction');
 const AuthenticationMethod = require('../../models/AuthenticationMethod');
 const moment = require('moment');
 
 function createAccessToken(userId) {
   const expirationDelaySeconds = settings.poleEmploi.accessTokenLifespanMs / 1000;
-  return jsonwebtoken.sign({ user_id: userId, source: POLE_EMPLOI }, settings.authentication.secret, {
-    expiresIn: expirationDelaySeconds,
-  });
+  return jsonwebtoken.sign(
+    {
+      user_id: userId,
+      source: constants.source.POLE_EMPLOI,
+      identityProvider: constants.identity_provider.POLE_EMPLOI,
```

Comment:
Du coup c'est ça qu'il faut en minuscule, le nom dans le token 
```suggestion
      identity_provider: constants.IDENTITY_PROVIDER.POLE_EMPLOI,
```